go.mod

```diff
@@ -26,10 +20,10 @@ require (
 	github.com/ipfs/go-ipfs-blockstore v0.0.1
 	github.com/ipfs/go-ipfs-chunker v0.0.1
 	github.com/ipfs/go-ipfs-cmdkit v0.0.1
-	github.com/ipfs/go-ipfs-cmds v0.0.1
+	github.com/ipfs/go-ipfs-cmds v0.0.5
 	github.com/ipfs/go-ipfs-exchange-interface v0.0.1
 	github.com/ipfs/go-ipfs-exchange-offline v0.0.1
-	github.com/ipfs/go-ipfs-files v0.0.1
+	github.com/ipfs/go-ipfs-files v0.0.2
```

Comment:
This is a surprisingly large diff and these version bumps look independent? To the extent that it's reasonable, please land these upgrades separately.

Reply:
This kept being updated after adding go.mod and go.sum in tools/migration, and a tag for the go-filecoin/tools/migration module.  I'll try to separate it as much as possible.

Reply:
✔️